src/fides/api/service/pagination/pagination_strategy_cursor.py

```diff
@@ -41,6 +47,14 @@ def get_next_request(
         if cursor is None:
             return None
 
+        if self.has_next and has_next is None:
+            has_next = pydash.get(response.json(), self.has_next)
+
+        if self.has_next:
+            logger.info(f"The {self.has_next} field has a value of {has_next}")
```

Comment:
We should make this a debug statement
```suggestion
            logger.debug(f"The {self.has_next} field has a value of {has_next}")
```